### PR TITLE
[Not merge] Use consul and guice bundles in user preference

### PR DESCRIPTION
**Pull request by sqreek**
This PR use ConsulBundle and PandoraGuiceBundle in User Preference

it's example and usage of changes made in https://github.com/Wikia/pandora/pull/532

### CODE REVIEW

File: service/user-preference/src/main/java/com/wikia/userpreference/UserPreferenceApplication.java [hunk 1]
```diff
@@ -3,48 +3,42 @@
...
-
   @SuppressWarnings("unchecked")
   @Override
   public void initialize(Bootstrap<UserPreferenceConfiguration> bootstrap) {
     super.initialize(bootstrap);
 
+    bootstrap.addBundle(new ConsulBundle(getName()));
+    bootstrap.addBundle(new PandoraGuiceBundle<>(
+        UserPreferenceApplication.class,
+        (builder -> {
```
Comment (WikiaTech) [inline service/user-preference/src/main/java/com/wikia/userpreference/UserPreferenceApplication.java]: MAJOR Remove those useless parentheses. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by pchojnacki** [comment]
Looks GOOD!

**Comment by drsnyder** [comment]
lgtm

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**